docs/troubleshooting.asciidoc

```diff
@@ -49,15 +49,27 @@ kibana-sample-kb-http          ClusterIP   10.19.246.116   <none>        5601/TC
 [id="{p}-describe-failing-resources"]
 === Describe failing resources
 
-If an Elasticsearch node does not come up, it's probably because Kubernetes cannot schedule the associated Pod. Check the Pod status: if it's not `Running` after a few seconds, something is wrong:
+If an Elasticsearch node does not come up, it is probably because Kubernetes cannot schedule the associated Pod.
+
+Start by looking at the StatefulSets to check that the current number of replicas match the desired number of replicas.
```

Comment:
Check the StatefulSets to see if the current number of replicas match the desired number of replicas.